README.md

```diff
@@ -201,6 +201,17 @@ app.startServer({
 });
 ```
 
+You can run your own command as well:
+
+```js
+app.startServer({
+  command: 'foo',
```

Comment:
Might be worth explicitly noting that this then gets run as `ember foo` as it isn't immediately apparent